lib/app.ts

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2016 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Import most things with types if possible.
+import Opts = require('node-getopt');
```

Comment:
Actually, I now remember why. It's because the `node-getopt` declarations declare using the `export = ` format and not a namespace. Therefore attempting to 'import ... from 'node-getopt'` fails, as it doesn't resolve to a module. Keeping as-is.